.ci/static-checks.sh

```diff
@@ -502,7 +549,7 @@ check_docs()
 		exit 1
 	fi
 
-	rm -f "$url_map" "$invalid_urls"
+	rm -f "$url_map" "$invalid_urls" "$md_links"
```

Comment:
Normally I'd ask for the `rm`s of tmpfiles to also be caught with a TRAP for the case when we do die. I'll let you decide if that is worth it or not.

Reply:
A few notes:

- This change is simply adding an extra file to the existing list.
- All the check functions use local variables and do their own cleanup currently.
- Retaining temporary files if this script fails is useful for debugging the failure, so the current behaviour is pretty sane I think.
- I agree that it might be useful to move the `rm` calls to a generic trap handler so that those files will be deleted if the script is successful, but retained on error (the trap handler could list the files to further help with debugging).

I've raised https://github.com/kata-containers/tests/issues/1613 so we can deal with that issue on another PR.